packages/shared/tailwind.config.js

```diff
@@ -186,24 +201,30 @@ module.exports = {
       gap: {
         unset: 'unset',
       },
-      borderRadius: {
-        2: '0.125rem',
-        3: '0.1875rem',
-        6: '0.375rem',
-        8: '0.5rem',
-        10: '0.625rem',
-        12: '0.75rem',
-        14: '0.875rem',
-        16: '1rem',
-        18: '1.125rem',
-        22: '1.375rem',
-        24: '1.5rem',
-        26: '1.625rem',
-        32: '2rem',
-        40: '2.5rem',
-        48: '3rem',
-        50: '3.125rem',
-      },
+      borderRadius: removeDefaults(
```

Comment:
We should probably not use extend, but overwrite then you don't need to remove defaults :)

Reply:
I see, I can't seem to find the docs for overwrite, do you maybe know a sample of how to do it?

Reply:
Ahh, I see it now. It is under the `extend` object.

Reply:
Yeah: https://tailwindcss.com/docs/theme#overriding-the-default-theme